Aptoide/aptoide-client-v8
# [MOB-149] Added adult content switch to no results on search;

**What does this PR do?**

This PR implements the new adult content flow when there are no results on search;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] SearchResultPresenter.java

**How should this be manually tested?**

Do adult content related searches with the preference set to off. Turn it on on the no results screen and make sure everything works as intended. Make sure to check the flow if a pin is set on adult content;

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-149](https://aptoide.atlassian.net/browse/MOB-149)

**Code Review Checklist**
